code/modules/power/tesla/energy_ball.dm

```diff
@@ -129,6 +129,10 @@
 /obj/singularity/energy_ball/Bumped(atom/movable/AM)
 	dust_mobs(AM)
 
+/obj/singularity/energy_ball/attack_tk(mob/user)
+	to_chat(user, "<span class='userdanger'>That was a really dumb idea.</span>")
```

Comment:
Strongly suggest `That was shockingly dumb.` or something along those lines that has some sort of dumb shock/electrocution wordplay in it.

Reply:
Oh man, I gotta. check next commit